modules/geoipbackend/geoipbackend.cc

```diff
@@ -51,9 +52,9 @@ struct GeoIPService
 
 struct GeoIPDomain
 {
-  int id;
+  std::size_t id{};
```

Comment:
Is this OK?

Reply:
it needs to fit in 31 bits, so uint32_t might make more sense (that's what dnsbackend.hh uses)